### PR TITLE
added a student card component which shows weekly meetings

### PR DESCRIPTION
**Pull request by Johng117**
Description:- I have added a card component to the student component and the endpoint which returns the users meeting for the next week. I have also added the Luxon package to parse dates and times.

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have carefully reviewed my own code
- [ ] I have commented my code
- [ ] I have updated any documentation

### CODE REVIEW

File: client/src/components/StudentCard.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+                    {/* optional heading, this can be changed */}
+					{cardData.length > 1 ? `You have ${cardData.length} Meetings`: "You have 1 Meeting"}
+				</Typography>
+			</Paper>
+
+			{cardData.map((element) => {
+				return (
+					<Card key={element.index} sx={{ mt: 2 }}>
+						<CardContent>
+							<Typography variant="h6">
```
Comment (TomRafferty) [inline client/src/components/StudentCard.js:60]: I think each of these need a key too. That should get rid of that error

**Review by TomRafferty — CHANGES_REQUESTED** [review]
Other than that key issue this looks awesome! Brilliant work John!

Changes:
-Add keys

File: client/src/components/StudentCard.js (hunk 1 above)
Comment (Johng117) [reply client/src/components/StudentCard.js:60]: I will have a look at it.

**Review by Maria-Chira — APPROVED** [review]
Looks good :)

**Review by TomRafferty — APPROVED** [review]
LGTM
great work John 😎

**Pull request merged**